examples/qtmegachatapi/chatItemWidget.cpp

```diff
@@ -177,11 +177,26 @@ ChatWindow *ChatItemWidget::showChatWindow()
     return mChatWindow;
 }
 
+bool ChatItemWidget::isChatOpened()
+{
+    return (mChatWindow ?true :false);
```

Comment:
Style correction: `mChatWindow ? true : false`
But actually you can directly do: `return mChatWindow;` The implicity casting will take place and it's 100% safe. A pointer to null evaluates as false